lib/acme_server/jobs/http_verifier.ex

```diff
@@ -1,23 +1,49 @@
 defmodule AcmeServer.Jobs.HttpVerifier do
+  @moduledoc false
```

Comment:
Curious why the docs below are not exposed as moduledocs

Reply:
We should use moduledoc only for modules and functions used by library users.